composer.json

```diff
@@ -28,9 +28,9 @@
         "codeception/lib-innerbrowser": "^3.0 || ^1.1",
         "codeception/module-asserts": "^3.0 || ^1.1",
         "codeception/module-yii2": "^1.1",
-        "codeception/module-filesystem": "^2.0 || ^1.1",
+        "codeception/module-filesystem": "^3.0 || ^1.1",
```

Comment:
Is Codeception alright w/ this major version bump?

Reply:
I have tested several machines with PHP 8.1-8.2, I got error 70% in previous machine

Reply:
Error 70%?

Reply:
> Error 70%?

When I am trying with "codeception/module-filesystem": "^2.0 || ^1.1",

Reply:
Yes, but what is "Error 70%?"?

Reply:
> 

If I tested on 6 machines(pc/os) with the previous version then out of 6 it error on 4 machines(pc/os) for the previous version & that's why I have said 70% error